app/policies/permissions.rb

```diff
@@ -121,7 +121,6 @@ module Permissions
         view_health_worker_activity
         download_patient_line_list
         manage_admins
-        view_my_facilities
```

Comment:
Why are we reverting this change?

Reply:
This wasn't intentional, I think this PR itself shouldn't add any permissions. I'll add that separately.

Reply:
Ah I see. This one will set up the script, and a subsequent PR will add this permission, and after that we'll run the script. 👍